include/seqan3/alphabet/quality/concept.hpp

```diff
@@ -127,10 +244,37 @@ SEQAN3_CONCEPT QualityAlphabet = requires(q quality)
 {
     requires Alphabet<q>;
 
-    { assign_phred(quality, typename q::rank_type{}) } -> q;
     { to_phred(quality) } -> const typename q::phred_type;
     typename underlying_phred<q>::type;
 };
 //!\endcond
 
-} // namespace seqan
+// ============================================================================
+// seqan3::WritableQualityAlphabet
+// ============================================================================
+
+/*!\interface seqan3::QualityAlphabet <>
+ * \extends seqan3::Alphabet
+ * \brief A concept that indicates whether a writable alphabet represents quality scores.
+ * \ingroup quality
+ *
+ * In addition to the requirements for seqan3::WritableAlphabet, the seqan3::WritableQualityAlphabet
+ * introduces the requirements of seqan3::QualityAlphabet.
+ *
+ * \par Concepts and doxygen
+ * The requirements for this concept are given as related functions and
+ * metafunctions. Types that satisfy this concept are shown as "implementing
+ * this interface".
+ */
+//!\cond
+template<typename q>
+SEQAN3_CONCEPT WritableQualityAlphabet = requires(q quality)
+{
+    requires WritableAlphabet<q>;
+    requires QualityAlphabet<q>;
```

Comment:
please require these "outside", because otherwise this alphabet will not be seen as a refinement in C++20.